app/services/SessionService.scala

```diff
@@ -100,6 +100,31 @@ class SessionService @Inject()(
       uiJourneySessionDataRepository.set(uiJourneySessionData)
   }
 
+  def setMultipleMongoData(kv: Map[String, String], journeyType: JourneyType)
```

Comment:
i think this method would return Future(Right(false)) if kv is empty. Would this be ok?